docs/griptape-framework/drivers/image-query-drivers.md

```diff
@@ -0,0 +1,30 @@
+# Image Query Drivers
+
+Image Query Drivers are used by [Image Query Engines](../engines/image-query-engines.md) to execute natural language queries on the contents of images. You can specify the provider and model used to query the image by providing the Engine with a particular Image Query Driver.
+
+## OpenAiVisionImageQueryDriver
+
+The [OpenAiVisionImageQueryDriver](../../reference/griptape/drivers/image_query/openai_vision_image_query_driver.md) is used to query images using the OpenAI Vision API. Here is an example of how to use it:
```

Comment:
`Here is an example of how to use it:` this should probably be placed right before the example.